src/content/development/contribution-guide/_index.en.md

```diff
@@ -46,7 +46,12 @@ To be expanded
 
 ## Release Cycles
 
-To be expanded
+The Submariner project follows time based release cycles, each cycle is 16 weeks long.
+Blocking bugs might cause a delay in the final release date, while new features will never be a reason to delay the release.
```

Comment:
```suggestion
Blocking bugs may delay final releases. New features shall not be a reason to delay releases.
```

Reply:
We're talking about the GA yes? So "releases" doesn't make sense in this context.

Also "shall not" sounds a bit too pompous IMHO :)